javaslang/src/test/java/javaslang/collection/euler/Euler24Test.java

```diff
@@ -1,76 +0,0 @@
-/*     / \____  _    _  ____   ______  / \ ____  __    _______
```

Comment:
Mmhh, the content is marked as deleted. What happens if I merge it? Will @jorander 's solution for Euler24Test disappear? Maybe you should close this PR and create a new PR containing only Euler21Test.

I hope that is ok for you, sorry for the inconvenience!

Greets,

Daniel